### PR TITLE
More accurate steps to use aws instance

### PR DESCRIPTION
**Pull request by xiaomaogy**

### CODE REVIEW

File: docs/Training-on-Amazon-Web-Service.md [hunk 1]
```diff
@@ -187,11 +187,32 @@ linux executables which use visual observations.
...
+Headless Mode, you have to setup the X Server to enable the training).
 5. Click Build to build the Unity environment executable.
 6. Upload the executable to your EC2 instance within `ml-agents` folder.
-7. Test the instance setup from Python using:
+7. Change permission on your executable.
+
+    ```console
+    chmod +x <your_env>.x86_64
+    ```
+8. (Without Headless Mode) Start X Server and make the ubuntu use X Server for
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: change to ... Start X Server and use it for display

File: docs/Training-on-Amazon-Web-Service.md [hunk 2]
```diff
@@ -202,9 +223,8 @@ linux executables which use visual observations.
     Where `<your_env>` corresponds to the path to your environment executable.
 
     You should receive a message confirming that the environment was loaded successfully.
-8. Train the executable
+10. Train the executable
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: Run the executable OR Train your models

File: docs/Training-on-Amazon-Web-Service.md [hunk 3]
```diff
@@ -187,11 +187,32 @@ linux executables which use visual observations.
...
+7. Change permission on your executable.
+
+    ```console
+    chmod +x <your_env>.x86_64
+    ```
+8. (Without Headless Mode) Start X Server and make the ubuntu use X Server for
+display:
+
+    ```console
+    //Start the X Server, press Enter to come back to the command line
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: isn't # more correct than //? See same for other lines.

File: docs/Training-on-Amazon-Web-Service.md [hunk 4]
```diff
@@ -187,11 +187,32 @@ linux executables which use visual observations.
...
-3. Select Linux as the Target Platform, and x86_64 as the target architecture.
-4. Check Headless Mode (If you haven't setup the X Server).
+3. Select Linux as the Target Platform, and x86_64 as the target architecture 
+(the default x86 currently doesn't work).
+4. Check Headless Mode (If you haven't setup the X Server, if you don't use
+Headless Mode, you have to setup the X Server to enable the training).
 5. Click Build to build the Unity environment executable.
 6. Upload the executable to your EC2 instance within `ml-agents` folder.
-7. Test the instance setup from Python using:
+7. Change permission on your executable.
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: Change the permissions of the executable.

File: docs/Training-on-Amazon-Web-Service.md [hunk 5]
```diff
@@ -187,11 +187,32 @@ linux executables which use visual observations.
 1. In the Unity Editor, load a project containing an ML-Agents environment (you
    can use one of the example environments if you have not created your own).
 2. Open the Build Settings window (menu: File > Build Settings).
-3. Select Linux as the Target Platform, and x86_64 as the target architecture.
-4. Check Headless Mode (If you haven't setup the X Server).
+3. Select Linux as the Target Platform, and x86_64 as the target architecture 
+(the default x86 currently doesn't work).
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: doesn't --> does not

File: docs/Training-on-Amazon-Web-Service.md [hunk 6]
```diff
@@ -187,11 +187,32 @@ linux executables which use visual observations.
...
+4. Check Headless Mode (If you haven't setup the X Server, if you don't use
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: "Check Headless Mode (If you haven't setup the X Server, if you don't use Headless Mode, you have to setup the X Server to enable the training)" --> "Check Headless Mode if you have not setup the X Server. (If you do not use Headless Mode, you have to setup the X Server to enable training)"

File: docs/Training-on-Amazon-Web-Service.md [hunk 7]
```diff
@@ -7,10 +7,10 @@ Service for training ML-Agents environments.
 
 We've prepared an preconfigured AMI for you with the ID: `ami-18642967` in the
 `us-east-1` region. It was created as a modification of [Deep Learning AMI
-(Ubuntu)](https://aws.amazon.com/marketplace/pp/B077GCH38C). If you want to do
-training without the headless mode, you need to enable X Server on it. After
-launching your EC2 instance using the ami and ssh into it, run the following
-commands to enable it:
+(Ubuntu)](https://aws.amazon.com/marketplace/pp/B077GCH38C). We've tested it
```
Comment (mmattar) [inline docs/Training-on-Amazon-Web-Service.md]: Change to:

The AMI has been tested with p2.xlarge instance. Furthermore, if you want to train without headless mode, you need to enable X Server. 

After launching your EC2 instance using the ami and ssh into it, run the following commands to enable it:

**Comment by xiaomaogy** [comment]
Hi @mmattar, thanks for the comments! I've resolved all of them. I know the comments should be marked '#' in bash script, but in this doc '//' is maybe a little bit more clear than '#'.

**Pull request merged**